test/scala/models/BusinessSearchRequestTest.scala

```diff
@@ -0,0 +1,30 @@
+package scala.models
+
+import models.BusinessSearchRequest
+import org.scalatest.{ FreeSpec, Matchers }
+import org.scalamock.scalatest.MockFactory
+import play.api.mvc.{ AnyContent, Request }
+
+class BusinessSearchRequestTest extends FreeSpec with Matchers with MockFactory {
```

Comment:
scalatest convention is to name such things Specs - you are obviously a big JUnit fan ;-)
(I'm sure we can live with Test.  It would be preferable if the codebase used a single convention however).